cypress/e2e/addVariantToCart.cy.ts

```diff
@@ -1,23 +1,22 @@
-import { PRODUCT_ELEMENTS } from "../elements/product-page";
 import { productsToSearch } from "../fixtures/search";
 import { addItemToCart, openProductPage } from "../support/pages/product-page";
 
 describe("Select variant and add to cart", () => {
   beforeEach(() => {
-    cy.visit("/");
+    cy.visit("/").clearLocalStorage();
   });
 
   it("should select a variant and add to the cart SRS_0202", () => {
     const product = productsToSearch.productWithVariants;
 
     openProductPage(product);
-    addItemToCart(PRODUCT_ELEMENTS.productName);
+    addItemToCart();
```

Comment:
It would be nice if at the end we check the cart icon - there should be a number of items in the cart

Reply:
In the future, there will be a mini cart and then the test will be checking if that product is in the cart.